src/Resources/ResourceLockResource.php

```diff
@@ -17,7 +17,12 @@ class ResourceLockResource extends Resource
 
     public static function getModel(): string
     {
-        return config('resource-lock.models.ResourceLock', ResourceLock::class);
+        return __(config('resource-lock.models.ResourceLock', ResourceLock::class));
```

Comment:
Looks good. Just out of curiosity, why did you use a translation function for the getModel?

Reply:
My mistake, I will fix that!